kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/embedder/MavenProjectLoader.java

```diff
@@ -38,7 +45,10 @@
 public class MavenProjectLoader {
 
     public static final String FORCE_OFFLINE = "kie.maven.offline.force";
+    public static final String RESOLVE_TRANSITVE = "kie.maven.offline.transitive.resolve";
+
     protected static boolean IS_FORCE_OFFLINE = Boolean.valueOf(System.getProperty(FORCE_OFFLINE, "false"));
+    protected static boolean IS_RESOLVE_TRANSITIVE = Boolean.valueOf(System.getProperty(RESOLVE_TRANSITVE, "false"));
```

Comment:
Hello Jan, what if I simply rename the static field?

I can't make it instance because it is used in static methods.

Indeed, it is protected for testing purpose!

Reply:
it's in sync with the line above, so unless we want to do wider refactoring, we can probably keep it as it is.